src/transaction/atomic_replication_helper.cpp

```diff
@@ -132,6 +132,25 @@ namespace cublog
     return atomic_sequence.can_end_atomic_sequence (sysop_parent_lsa);
   }
 
+  log_lsa atomic_replication_helper::get_the_lowest_start_lsa ()
+  {
+    log_lsa min_lsa = MAX_LSA;
+
+    for (auto const &sequence_map_iterator : m_sequences_map)
+      {
+	if (sequence_map_iterator.second.get_start_lsa () < min_lsa)
+	  {
+	    min_lsa = sequence_map_iterator.second.get_start_lsa ();
+	  }
+      }
+    return min_lsa;
+  }
+
+  bool atomic_replication_helper::is_any_atomic_sequence_open () const
```

Comment:
No need to have this function.
If there are no sequences open, `get_the_lowest_start_lsa` can answer with a guard value - `MAX_LSA` and the caller will reconcile with the value from the helper (either valid value or guard value) with the one from the atomic replicator.